homeassistant/components/weatherflow_cloud/weather.py

```diff
@@ -0,0 +1,145 @@
+        coordinator: WeatherFlowCloudDataUpdateCoordinator,
+        station_id: int,
+    ) -> None:
+        """Initialise the platform with a data instance and station."""
+        super().__init__(coordinator)
+
+        self.station_id = station_id
+        self._attr_unique_id = f"weatherflow_forecast_{station_id}"
+
+        # Obtain info from 1st outdoor device on station
+        outdoor_device = self.local_data.station.outdoor_devices[0]
+
+        self._attr_device_info = DeviceInfo(
+            name=self.local_data.station.name,
+            entry_type=DeviceEntryType.SERVICE,
+            identifiers={(DOMAIN, outdoor_device.serial_number)},
+            manufacturer=MANUFACTURER,
+            configuration_url=f"https://tempestwx.com/station/{station_id}/grid",
+            serial_number=outdoor_device.serial_number,
+            sw_version=outdoor_device.firmware_revision,
+            hw_version=outdoor_device.hardware_revision,
+        )
+
+    @property
+    def local_data(self) -> WeatherFlowData:
+        """Return the local weather data object for this station."""
+        return self.coordinator.data[self.station_id]
+
+    @property
+    def condition(self) -> str | None:
+        """Return current condition - required property."""
+        return self.local_data.weather.current_conditions.icon.ha_icon
+
+    @property
+    def native_temperature(self) -> float | None:
+        """Return the temperature."""
+        return self.local_data.weather.current_conditions.air_temperature
+
+    @property
+    def native_pressure(self) -> float | None:
+        """Return the Air Pressure @ Station."""
+        return self.local_data.weather.current_conditions.station_pressure
+
+    #
+    @property
+    def humidity(self) -> float | None:
+        """Return the humidity."""
+        return self.local_data.weather.current_conditions.relative_humidity
+
+    @property
+    def native_wind_speed(self) -> float | None:
+        """Return the wind speed."""
+        return self.local_data.weather.current_conditions.wind_avg
+
+    @property
+    def wind_bearing(self) -> float | str | None:
+        """Return the wind direction."""
+        return self.local_data.weather.current_conditions.wind_direction
+
+    @property
+    def native_wind_gust_speed(self) -> float | None:
+        """Return the wind gust speed in native units."""
+        return self.local_data.weather.current_conditions.wind_gust
+
+    @property
+    def native_dew_point(self) -> float | None:
+        """Return dew point."""
+        return self.local_data.weather.current_conditions.dew_point
+
+    @property
+    def uv_index(self) -> float | None:
+        """Return UV Index."""
+        return self.local_data.weather.current_conditions.uv
+
+    @callback
+    def _async_forecast_daily(self) -> list[Forecast] | None:
+        """Return the daily forecast in native units."""
+        return [x.ha_forecast for x in self.local_data.weather.forecast.daily]
```

Comment:
As a general remark the library's should not have HA specific things including naming.